docs/examples/userguide/memoryviews/slicing.pyx

```diff
@@ -0,0 +1,10 @@
+import numpy as np
+
+exporting_object = np.empty((15, 10, 20), dtype=np.intc)

```

Comment:
I wouldn't use `np.empty()` here, since it creates an uninitialised buffer, and we are reading from it below.